### DISCUSSION TITLE
Add "double click username to reply" function like in FFZ

### DISCUSSION THREAD

**Discussion by dubbyOW** [SevenTV/Extension#546] [Ideas]
Add "double click username to reply" function like in FFZ Enhancing Add-On, I really like this function, but it doesn't work anymore if 7TV is installed on top of FFZ.
![image](https://user-images.githubusercontent.com/119939110/235051515-b2049d99-a182-4ba5-8ac3-c86b85bfba6e.png)

what it does: double click username in chat and it automatically puts their username with a comma in chat input box

**Comment by ayyybubu** [comment]
#521
